### ISSUE TITLE
[table] Screenreader users unable to determine associate table content with heading

### ISSUE THREAD

**Issue by UsmanAfzal** [WeAreSnook/OR-UK-frontend#186]
URL: https://openreferraluk.org/dashboard

The table headings require the scope attribute with the value "col" so that screenreader users are informed of table content  its association to which heading.

<img width="940" alt="Screenshot 2021-06-29 at 13 58 10" src="https://user-images.githubusercontent.com/2459151/123801292-0d2a0c00-d8e2-11eb-961e-05c263799149.png">

Current HTML:
```
<thead>
  <tr>
    <th>
      <div class="with-help">
        Organisation
        <div class="help" tabindex="0">
          <div class="help-text">
            <section class="">
              <p>Name of the organisation publishing services data</p>
            </section>
          </div>
        </div>
      </div>
    </th>
</thead>
```

Update HHTML (second line update):
```
<thead>
  <tr scope="col">
    <th>
      <div class="with-help">
        Organisation
        <div class="help" tabindex="0">
          <div class="help-text">
            <section class="">
              <p>Name of the organisation publishing services data</p>
            </section>
          </div>
        </div>
      </div>
    </th>
</thead>
```

**Comment by kingdomlevel** [comment]
Dashboard-related issue. Depends on #209 being merged before this can be fixed! Will revisit once #209 has been merged 👍

**Comment by kingdomlevel** [comment]
PR #209 has now been merged - removing the 'blocked' label and moving to 'To Do'

**Comment by kingdomlevel** [comment]
Getting the following warning in trying to implement the above suggested markup:

![image](https://user-images.githubusercontent.com/20354076/126756353-d85225d1-6055-4f43-9b09-e29df54f36ea.png)

...saying the scope attribute on `<tr>` is disallowed:
- [.eslint rule](https://github.com/jsx-eslint/eslint-plugin-jsx-a11y/blob/master/docs/rules/scope.md)
- [WCAG 1.3.1](https://www.w3.org/WAI/WCAG21/Techniques/html/H63).

---

I'm going to implement the markup with `scope` attribute on each header cell for now:
```html
<thead>
      <tr>
        <th scope="col">
          <Title help={help}>Organisation</Title>
        </th>
        <th scope="col">
          <Title help={help}>Developer</Title>
        </th>
        <th scope="col">
          <Title help={help}>Endpoint up</Title>
        </th>
        <th scope="col">
          <Title help={help}>Services feed</Title>
        </th>
        <th scope="col">
          <Title help={help}>{`Service {id} feed`}</Title>
        </th>
        <th scope="col">
          <Title help={help}>Searchable</Title>
        </th>
        <th scope="col">
          <Title help={help}>Last checked</Title>
        </th>
        <th scope="col">
          <Title help={help}>Summary</Title>
        </th>
        <th scope="col">
          <Title help={help}>Explore</Title>
        </th>
      </tr>
    </thead>
```

@UsmanAfzal  - let me know if this isn't an acceptable and I'll be more than happy to change! Thanks for all your work on these tickets so far 💪

**Comment by UsmanAfzal** [comment]
Oops @kingdomlevel that's correct, always meant to be on the `<th>`.

**Comment by UsmanAfzal** [comment]
@adamgparsons @kingdomlevel The data on the dashboard does not show up on the test site https://or-uk-frontend.herokuapp.com/, will this only be available when live?

**Comment by UsmanAfzal** [comment]
checked and confirmed
